src/azure-cli/requirements.py3.Linux.txt

```diff
@@ -31,7 +31,7 @@ azure-mgmt-compute==10.0.0
 azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==1.5.0
 azure-mgmt-containerregistry==3.0.0rc8
-azure-mgmt-containerservice==8.1.0
+azure-mgmt-containerservice==8.2.0
```

Comment:
Darwin and Windows missed ?

Reply:
@arrownj this was needed before it was merged to the python sdk. Now this change is gone.